.goreleaser.yml

```diff
@@ -0,0 +1,111 @@
+# For more about how to configure goreleaser, see:
+# https://goreleaser.com/customization/
+#
+# To release:
```

Comment:
do you think we'll plan on creating release branches as well, or just do all releases from tags on commits in master?

Reply:
We probably don't need the release branches anymore, but we might still create them because they're useful if we ever want to make a patch for an old release while we're working on a new release.